File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/orderverify/d/OrderVerifyDepDTO.java
package iih.ci.ord.dto.orderverify.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医嘱审核病区 DTO数据 
 * 
 */
public class OrderVerifyDepDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 病区id
	 * @return String
	 */
	public String getId_dep() {
		return ((String) getAttrVal("Id_dep"));
	}
	/**
	 * 病区id
	 * @param Id_dep
	 */
	public void setId_dep(String Id_dep) {
		setAttrVal("Id_dep", Id_dep);
	}
	/**
	 * 病区名称
	 * @return String
	 */
	public String getName_dep() {
		return ((String) getAttrVal("Name_dep"));
	}
	/**
	 * 病区名称
	 * @param Name_dep
	 */
	public void setName_dep(String Name_dep) {
		setAttrVal("Name_dep", Name_dep);
	}
	/**
	 * 医嘱数量
	 * @return Integer
	 */
	public Integer getOrder_num() {
		return ((Integer) getAttrVal("Order_num"));
	}
	/**
	 * 医嘱数量
	 * @param Order_num
	 */
	public void setOrder_num(Integer Order_num) {
		setAttrVal("Order_num", Order_num);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/orderverify/d/OrderVerifyDepDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.orderverify.d
{
    /// <summary>
    /// OrderVerifyDepDTO 的摘要说明。
    /// </summary>
    public class OrderVerifyDepDTO : BaseDTO {

        public OrderVerifyDepDTO() {
 
        }

        /// <summary>
        /// 病区id
        /// </summary>
		public string Id_dep {
            get { return getAttrVal<string>("Id_dep",null); }
            set { setAttrVal<string>("Id_dep", value); }
        }

        /// <summary>
        /// 病区名称
        /// </summary>
		public string Name_dep {
            get { return getAttrVal<string>("Name_dep",null); }
            set { setAttrVal<string>("Name_dep", value); }
        }

        /// <summary>
        /// 医嘱数量
        /// </summary>
		public int? Order_num {
            get { return getAttrVal<int?>("Order_num",null); }
            set { setAttrVal<int?>("Order_num", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_dep", "Name_dep", "Order_num"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.orderverify.d.OrderVerifyDepDTO";
        }
    }
}
